rotkehlchen/chain/ethereum/names.py

```diff
@@ -205,6 +210,12 @@ def _ens_address_to_name(
         chain_address: OptionalChainAddress,
 ) -> Optional[str]:
     """Returns the ens name for an address or None if the address doesn't have one"""
+    if (
+        chain_address.blockchain is None or
+        chain_address.blockchain != SupportedBlockchain.ETHEREUM
+    ):
+        return None  # ENS names are only for ethereum
```

Comment:
Why though? So long as the address is the same and ENS registrar exists only in ethereum shouldn't this work the same?

I am not sure about this. Need to think a bit.